README.md

```diff
@@ -44,35 +53,87 @@ Run `ClientApp.main()` to start the client and see it connecting to the server.
-the following steps are suggested:
-
-### Gradle Configuration Changes:
-
- * Update `*.gradle` files with the artifact attributes, that correspond to your project.
-
-### Suggested `model` Changes 
-
- * Define the domain, in which the business task is solved.
- * Together with the domain experts, perform event storming in the bounded context of the domain 
- speaking in language, ubiquitous for this domain.
- * According to the event storming results, define events, commands, entity states and value 
-objects in Protobuf. Put them into the `model/proto` folder. 
- * Design and implement Aggregates, Projections and Process Managers on top of the previously 
- defined language elements. Create repositories for them.
-
-### Suggested `server` Changes 
-
- * Configure and the storage factory, that corresponds to the target environment. Connectors to 
- [Google Datastore](https://github.com/SpineEventEngine/gcloud-java) and 
- [JDBC-enabled storages](https://github.com/SpineEventEngine/jdbc-storage) are provided by Spine.
- * Append the creation of the bounded context with the registration of newly created repositories.
- 
- Other possible changes include dealing with security (e.g. authentication/authorization), defining
- deployment scheme, scaling approach etc. These and other advanced topics aren't covered by this sample.
- 
-
-Typically these steps are repeated for each bounded context in the application.
- 
-For more information please visit [the official website](https://spine.io).
+ * Experiment with the model—create a new command type:
+```proto
+// -- commands.proto --
+
+message AssignDueDate {
+    
+    TaskId task_id = 1;
+    
+    google.protobuf.Timestamp due_date = 2 [(when).in = FUTURE];
+}
+```
+and a new event type:
+```proto
+// -- events.proto --
+
+message DueDateAssigned {
+    
+    TaskId task_id = 1;
+    
+    google.protobuf.Timestamp due_date = 2;   
+}
+```
+Remember to import `Timestamp` via `import "google/protobuf/timestamp.proto";`.
+ * Adjust the aggregate state:
+```proto
+message Task {
+    option (entity).kind = AGGREGATE;
+
+    // An ID of the task.
+    TaskId id = 1;
+
+
+    // A title of the task.
+    string title = 2 [(required) = true];
+    
+    // The date and time by which this task should be completed.
+    google.protobuf.Timestamp due_date = 3; // <-- New field
+}
+```
+Make sure to run a Gradle build after the changing the Protobuf definitions.
+ * Handle the `AssignDueDate` command in the `TaskAggregate`:
+```java
+@Assign
+DueDateAssigned handle(AssignDueDate command) {
+    return DueDateAssignedVBuilder
+            .newBuilder()
+            .setTaskId(command.getTaskId())
+            .setDueDate(command.getDueDate())
+            .build();
+}
+```
+ * Apply the emitted event:
+```java
+@Apply
+private void on(DueDateAssigned event) {
+    builder().setDueDate(event.getDueDate());
+}
+```
+ * In `ClientApp`, append the `main()` method. Post another command:
```

Comment:
Did you mean ”extend”? The word ”append” sounds as we're adding yet another method to the bottom of the class source code.